File: Problem Set 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    answer = get_int("Height: ")
    if answer > 0 and answer < 9:
        break

j = 1

for i in range(answer):
    # print the spaces before the first block
    print(" "*(answer-j),  end="")
    # print the left side of the pyramid
    print("#"*j,  end="")
    # print the space between the bricks
    print("  ", end="")
    # print the right side of the pyramid
    print("#"*j,  end="")
    # get new line
    print()
    # add 1 to j to 1 more brick
    j += 1

File: Problem Set 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_brick(int bricks, int spaces);

int main(void)
{
    int height;
    do
    {
        height = get_int("Please enter a height between 1-8: ");
    }
    while (height < 1 || height > 8);

    // main function to print bricks
    for (int i = 1; i <= height; i++)
    {
        print_brick(i, height);
        printf("\n");
    }
}


void print_brick(int bricks, int spaces)
{
    // Print the spaces before the bricks
    for (int k = spaces - bricks; k >= 1; k--)
    {
        printf(" ");
    }
    // Print the first column(s) of bricks
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }

    // print the space in between the bricks
    printf("  ");

    // print the second column(s) of bricks
    for (int j = 0; j < bricks; j++)
    {
        printf("#");
    }
}


